api-specs/Gateway-EE/3.5/kong-ee-3.5.yaml

```
@@ -10156,88 +11210,2472 @@ paths:
+                              description:
+                                type: string
+                              fields:
+                                type: array
+                                items:
+                                  type: string
+                              unique:
+                                type: array
+                                items:
+                                  type: string
+                x-examples:
+                  Example 1:
+                    data:
+                      balancer:
+                        health:
+                          fields:
+                            - upstream_id
+                            - ip
+                            - port
+                            - hostname
+                            - health
+                      crud:
+                        acls:
+                          fields:
+                            - operation
+                            - entity
+                            - old_entity
+                            - schema
+                      rate-limiting-advanced:
+                        rate-limit-exceeded:
+                          description: Run an event when a rate limit has been exceeded
+                          fields:
+                            - consumer
+                            - ip
+                            - service
+                            - rate
+                            - limit
+                            - window
+                          unique:
+                            - consumer
+                            - ip
+                            - service
+              examples:
+                sources example:
+                  value:
+                    data:
+                      balancer:
+                        health:
+                          fields:
+                            - string
+                      crud:
+                        acls:
+                          fields:
+                            - string
+                      rate-limiting-advanced:
+                        rate-limit-exceeded:
+                          description: string
+                          fields:
+                            - string
+                          unique:
+                            - string
+      operationId: get-event-hooks-sources
+      description: |
+        Sources are the actions that trigger the event hook. The /sources JSON output follows the following pattern:
+
+        * 1st level = The source, which is the action that triggers the event hook.
+        * 2nd level = The event, which is the Kong entity the event hook will listen to for events.
+        * 3rd level = The available template parameters for use in webhook-custom payloads.
+
+        For instance, in the example below balancer is the source, health is the event, and upstream_id, ip, port, hostname, and health are the available template parameters.
```

```suggestion
        Sources are the actions that trigger the event hook. The `/sources` JSON output follows the following pattern:

        * 1st level = The source, which is the action that triggers the event hook.
        * 2nd level = The event, which is the Kong entity the event hook listens to for events.
        * 3rd level = The available template parameters for use in `webhook-custom` payloads.

        For instance, in the example below `balancer` is the source, `health` is the event, and `upstream_id`, `ip`, `port`, `hostname`, and `health` are the available template parameters.
```
Is there an example below?